opm/input/eclipse/Deck/value_status.hpp

```diff
@@ -20,37 +20,28 @@
 #ifndef VALUE_STATUS
 #define VALUE_STATUS
 
-namespace Opm {
-
-namespace value {
-
-enum class status : unsigned char { uninitialized = 0,
-                                    deck_value = 1,
-                                    empty_default = 2,
-                                    valid_default = 3 };
-
-
-inline bool defaulted(status st) {
-    if (st == status::empty_default)
-        return true;
-
-    if (st == status::valid_default)
-        return true;
-
-    return false;
+namespace Opm { namespace value {
```

Comment:
why not just Opm::value ?

Reply:
> why not just Opm::value ?

Because I'm :older_man: and haven't internalised those new-fangled constructions...  I'll update.